dist/index.js

```diff
@@ -3148,15 +3148,15 @@ function kick({ adminClient, client, user, teams, owner, repo, issueNumber }, su
     return __awaiter(this, void 0, void 0, function* () {
         core.debug(`kicking subject: ${subject}`);
         const membershipResponse = yield adminClient.orgs.getMembership({ org: owner, username: user });
-        const canExecuteCommand = !(yield isTeamMember(adminClient, user, owner, teams)) || membershipResponse.data.role !== 'admin';
+        const canExecuteCommand = (yield isTeamMember(adminClient, user, owner, teams)) || membershipResponse.data.role === 'admin';
```

Comment:
triple equals? what's that?

Reply:
Its a javascript'ism meaning exactly equal to instead of kinda equal to.  
Double equals will try to massage types to see if they are equal and triple won't.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness